src/regex/lib_pcre.cr

```diff
@@ -23,5 +23,7 @@ lib LibPCRE
   $pcre_free : Free
 end
 
-LibPCRE.pcre_malloc = ->GC.malloc(LibC::SizeT)
-LibPCRE.pcre_free = ->GC.free(Void*)
+{% unless flag?(:without_pcre) %}
```

Comment:
These two lines are the ones responsible for *always* linking apps against PCRE into the built binary, whatever if the application uses PCRE or not. It may be interesting to have a special macro for `lib` that would be called whenever that `lib` is actually used @asterite 

I disable it because I don't need it (yet) and building libraries for the NDK is a pain, so I settled for `iconv`, `gc` and `libevent` for the time being.